tests/csv/3.0/mapped_joint_bam.fixed.csv

```diff
@@ -0,0 +1,3 @@
+patient,status,sample,bam,bai
```

Comment:
shouldn't you replace the old one?

Reply:
I decided to postpone that, since the change would probably require that I update a lot of md5sums. I just can't be bothered to do that now. sorry

https://github.com/nf-core/sarek/issues/872

Reply:
ok, good for me, no worries